vagrant-kubernetes-istio/RunTestOnHost/test_setup.sh

```diff
@@ -11,8 +11,8 @@ docker images -q |xargs docker rmi
 
 # Make and Push images to insecure local registry on VM.
 cd $ISTIO/istio
-make docker HUB=10.10.0.2:5000 TAG=latest
-make push HUB=10.10.0.2:5000 TAG=latest
+GOOS=linux USE_DOCKER=true make docker HUB=10.10.0.2:5000 TAG=latest
```

Comment:
I mean setting USE_DOCKER (or any other sensible name) in the push and docker commands directly, so user script (like test_setup.sh) does not have to set it.

Reply:
What is the disadvantage of setting the env in user script?